common/components/org.wso2.carbon.data.agents.log4j/src/main/java/org/wso2/carbon/data/agents/log4j/appender/LogEventAppender.java

```diff
@@ -301,23 +298,23 @@ private void publishLogEvent(TenantAwareLoggingEvent event)
                 dataPublisher = new DataPublisher("Thrift", url, authURLSet, userName, password);
             } catch (DataEndpointAgentConfigurationException e) {
                 log.error(
-                        "Error in initializing binary data-publisher to send requests to global throttling engine " + e
+                        "Invalid urls passed for receiver and auth, and hence expected to fail " + e
```

Comment:
Better if we know what causes each exception and log a relevant error so that end user can get a hint of what can be done to rectify.